public/docs/dart/latest/api/index.jade

```diff
@@ -1,10 +1,10 @@
-.l-main-section
-  h2 Beta
+:marked
+  > **Known issues:** Although this generated API reference displays Dart APIs
+  and includes some Dart examples, the text reflects the TypeScript APIs. The
+  angular.io issue tracker contains [all known issues][api-issues]; if you
+  notice others, please [report them][ngio-issues]. Thanks!
 
-  p.
-    The proposed Angular 2 API does not yet have Dart-specific documentation.
-    However, because the Dart and JavaScript APIs are generated from the same source,
-    you might find the JavaScript API docs helpful:
+  [ngio-issues]: https://github.com/angular/angular.io/issues
```

Comment:
How about this:
> https://github.com/angular/angular.io/issues/new?labels=dart,api&title=[Dart][API]

The title prefix is because not everyone has the permission to set the labels, so a title prefix is better than nothing.